tools/src/main/scala/scala/scalanative/interflow/Interflow.scala

```diff
@@ -20,44 +20,76 @@ class Interflow(val config: build.Config)(implicit
     with PolyInline
     with Intrinsics
     with Log {
+
+  /** The target machine for which code is being compiled. */
+  // QUESTION: Is it the host or the target.
+  // QUESTION: If the latter, shouldn't we rename this "target"?
```

Comment:
It's the target. Some of it's field like size of pointer can would depend on the explicit LLVM target triple passed in the config allowing for cross compilation. So it could be renamed to TargetInfo